### DISCUSSION TITLE
各位可以帮个忙吗? 您在付费的选项里是否能看到支付宝的选项吗? 如果在Alipay遇到什么问题请在这个帖子反馈.

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#769] [Announcements]
对了, 各位可以帮我个忙吗? 您在付费的选项里, 帮我查看有支付宝的选项吗? 我们在做测试. 想了解用户可以看到吗. 谢谢您!

同时您如果在Alipay付费后遇到什么问题请在这个帖子反馈. 我们会帮忙解决问题. 谢谢!

**Comment by lizihui12345678** [comment]
今天买的， 看不到，只有银行卡信用卡支付选项，填写完邮箱点击进入填写银行账户信息界面，出现有支付宝支付的选项，当天下午已使用支付宝扫码并支付成功。

**Comment by MaxMKMax** [comment]
@lizihui12345678 好的谢谢您的反馈. 我会帮忙反馈给支付团队和科技团队看怎么优化购买体验会更好.

**Reply by sobai14150** [reply]
没看到支付宝 支付选项 ，专业版本的激活码要如何购买？  去年购买激活码的网页失效， 有没有新购买渠道。

**Comment by ruochen800** [comment]
The email of the paid account received an eight-digit code: 75903989.

There is an authorization problem. I have tried every method suggested, but none of them worked. The payment was clearly very normal, but due to issues during payment, I tried different devices and accidentally deauthorized the original system. After making the payment, I could no longer authorize it. I no longer have any hope for this.

However, an extra 15% fee of 84 yuan was charged for two years, and it should have extended the service by 110 days. Can't you do that? Why hasn't this been resolved?

Please resolve this as soon as possible, thank you!

**Reply by ruochen800** [reply]
I really don't understand why everything was working smoothly before the renewal, but it stopped working after I renewed. I followed the methods I found online, and by creating a new Windows account, I finally succeeded in authorizing it. However, I can't share it with the original system, and using it directly on the new system is extremely difficult. It's almost unusable, with a single webpage often taking more than ten minutes to load.

**Comment by Ericqdhw** [comment]
https://github.com/getlantern/lantern-forum-cn/issues/723   你好 请解决一下 谢谢！！

**Comment by henrylyubj** [comment]
我刚才又试了一下，没有支付宝选项。最后银行卡还是不成功，显示为：”处理您的付款时发生了错误。请稍后再试或使用另一支付方式。“

**Comment by wlln157** [comment]
我需呀一个支付宝付费选项，银行卡一直通过不了，无法为账号付费。55385338@qq.com

**Comment by xwyqsl** [comment]
@Mike777Khoo 
您好
关于支付宝选项的问题 我的页面开始也是没有 后来升级到最新版本之后就出现支付宝选项了
我现在遇到的问题是
付款之后是一个国内的公司 名为 湖北明宝雄科技有限公司
付款大概两个小时了 还是没有同步更新 蓝灯页面依旧显示7天后到期
想问一下是怎么回事 我是被骗了吗 谢谢
订单编号：2024091122001473871410031996

**Comment by Honeydukes3** [comment]
我刚才用支付宝购买专业版后没有升级。账号：771324905donald@tutanota.com。
![1](https://github.com/user-attachments/assets/c227bbd8-6206-4a6d-b680-b6288d3b1d1b)

**Reply by Honeydukes3** [reply]
@Mike777Khoo
